src/main/kotlin/com/sourcegraph/cody/initialization/ActionUpdateThreadClassloaderPatcher.kt

```diff
@@ -1,26 +1,15 @@
 package com.sourcegraph.cody.initialization
 
 import com.intellij.openapi.application.ApplicationInfo
-import com.intellij.util.lang.ClassPath
 import com.intellij.util.lang.UrlClassLoader
-import kotlin.io.path.name
-import kotlin.reflect.full.memberFunctions
-import kotlin.reflect.jvm.javaMethod
 
 object ActionUpdateThreadClassloaderPatcher {
   init {
     val appInfo = ApplicationInfo.getInstance()
-    if (appInfo.majorVersion > "2022" || appInfo.minorVersion > "1") {
+    if (appInfo.majorVersion == "2022" && appInfo.minorVersion <= "1") {
```

Comment:
We don't support anything before 2022.1 right? Because otherwise this check wouldn't work

Reply:
We don't:
`<idea-version since-build="221.5080.210"/>`